micromanager_gui/main_window.py

```diff
@@ -125,6 +123,27 @@ def __init__(self, viewer: napari.viewer.Viewer, remote=False):
         self.viewer.layers.selection.events.active.connect(self.update_max_min)
         self.viewer.dims.events.current_step.connect(self.update_max_min)
 
+        @self.viewer.mouse_drag_callbacks.append
+        def get_event(viewer, event):
+            if not self.explorer.isVisible():
+                return
+            if self._mmc.getPixelSizeUm() > 0:
+                width = self._mmc.getROI(self._mmc.getCameraDevice())[2]
+                height = self._mmc.getROI(self._mmc.getCameraDevice())[3]
+
+                x = viewer.cursor.position[-1] * self._mmc.getPixelSizeUm()
+                y = viewer.cursor.position[-2] * self._mmc.getPixelSizeUm() * (-1)
+
+                # to match position coordinates with center of the image
+                x = f"{x - ((width / 2) * self._mmc.getPixelSizeUm()):.1f}"
+                y = f"{y - ((height / 2) * self._mmc.getPixelSizeUm() * (-1)):.1f}"
+
+            else:
+                x, y = "None", "None"
+
+            self.x_lineEdit.setText(x)
+            self.y_lineEdit.setText(y)
```

Comment:
```suggestion
            self.explorer.x_lineEdit.setText(x)
            self.explorer.y_lineEdit.setText(y)
```

Reply:
ah good catch. In a future PR we should decouple this from the main window. But that may be more complicated

Reply:
Unresolving so we don't lose track of this as a next step. Probably the explorer widget should have an explicit signal that whatever is displaying it  can use to update the explorer.